pyiron/testing/randomatomistic.py

```diff
@@ -461,6 +461,9 @@ def from_hdf(self, hdf=None, group_name=None):
         super(AtomisticExampleJob, self).from_hdf(hdf=hdf, group_name=group_name)
         self._structure_from_hdf()
 
+    def run_static(self):
```

Comment:
I do not think this change belongs to this pull request.

Reply:
That's true - now I'm going to open another pull request